commonui/src/main/res/values/strings.xml

```diff
@@ -24,4 +24,7 @@
 
     <string name="transfer_method_list_item_description">Ending on %s</string>
     <string name="not_translated_in_braces">(not translated)</string>
+
+    <string name="environment_prod">PROD</string>
+    <string name="environment_uat">UAT</string>
```

Comment:
Please remove this we don't need this resources anymore since we got it from Auth Config